easydita-knowledge-portal/sass/ezd/footer.scss

```diff
@@ -58,17 +58,20 @@
 	i.linkedin-icon:after {
 		content: "\f08c"
 	}
+  i.youtube-icon:after {
+		content: "\f167"
```

Comment:
Let's use \f166 as the default here (with the background: http://fontawesome.io/icon/youtube-square/) so that it matches the styling of the other icons in the default skin. Then, you can use \f167 in the skin1/footer.css file to overwrite the icon for the new skin.